### PR TITLE
Adds vox

### PR DESCRIPTION
**Pull request by SamHPurp**
Adds/removes a bunch of Vox announcements for the AI

Adds: active, airlock, alcohol, arrival, arrivals, between, blob, briefing, chair, chapel, closet, declared, detective, dock, docked, docking, docks, drunk, emag, equipment, evacuation, experimentor, gateway, hail, hallway, holoparasite, hull, imminent, intruders, library, mime, mimes, miner, miners, mining, n2o, nuke, office, operational, poly, pun, renault, runtime, scrubbers, space, spider, spiders, still, swarmer, swarmers, sword, technology, teleport, teleported, teleporter, teleporting, tesla, tool, unathi, vacant, vault, vents, weld, window, windows, yet

Removes: abortions, adios, amigo, ass, asshole, assholes, backman, bailey, birdwell,bitch, bitches, canal, centi, coded, connor, coomer, cunt, deeoo, ed, freeman, gordon, guthrie, helicopter, shitlord, shitting, ss13, voxtest, vox_login, wood, woody

:cl: Purpose2
tweak: Adds and removes a bunch of different AI words for the vox_fem announcement system.
/:cl:

### CODE REVIEW

**Comment by Xantholne** [comment]
Can't say I agree to removing all of those. But some could definitely be removed.

Ones I can't say I'd like see gone are asshole, assholes, bitch, bitches, deeoo,ss13, voxtest, vox_logiin

**Comment by Ziiro** [comment]
>Adds vox
>Vox announcements

***aflap**

Cid had a good idea to give all the worst offenders to malfunctioning AI only, which I kind of agree with.

**Comment by Xantholne** [comment]
Where would we be if we couldnt say that

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 1]
```diff
@@ -687,15 +720,16 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
...
 "west" = 'sound/vox_fem/west.ogg',
 "whiskey" = 'sound/vox_fem/whiskey.ogg',
 "white" = 'sound/vox_fem/white.ogg',
 "wilco" = 'sound/vox_fem/wilco.ogg',
 "will" = 'sound/vox_fem/will.ogg',
+"window" = 'sound/vox_fem/window.ogg',
+"windows" = 'sound/vox_fem/windows.ogg',
 "with" = 'sound/vox_fem/with.ogg',
 "without" = 'sound/vox_fem/without.ogg',
-"wood" = 'sound/vox_fem/wood.ogg',
```
Comment (FlattestGuitar) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm]: I get "woody" being gone, but this one?

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 2]
```diff
@@ -118,23 +115,21 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
 "clear" = 'sound/vox_fem/clear.ogg',
 "clearance" = 'sound/vox_fem/clearance.ogg',
 "close" = 'sound/vox_fem/close.ogg',
+"closet" = 'sound/vox_fem/closet.ogg',
 "clown" = 'sound/vox_fem/clown.ogg',
 "code" = 'sound/vox_fem/code.ogg',
-"coded" = 'sound/vox_fem/coded.ogg',
```
Comment (FlattestGuitar) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm]: What's wrong with "coded"?

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 3]
```diff
@@ -106,10 +102,11 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
 "celsius" = 'sound/vox_fem/celsius.ogg',
 "centcom" = 'sound/vox_fem/centcom.ogg',
 "center" = 'sound/vox_fem/center.ogg',
-"centi" = 'sound/vox_fem/centi.ogg',
```
Comment (FlattestGuitar) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm]: Why

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 4]
```diff
@@ -97,7 +94,6 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
 "cable" = 'sound/vox_fem/cable.ogg',
 "call" = 'sound/vox_fem/call.ogg',
 "called" = 'sound/vox_fem/called.ogg',
-"canal" = 'sound/vox_fem/canal.ogg',
```
Comment (FlattestGuitar) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm]: Why

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 5]
```diff
@@ -63,24 +62,21 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
 "away" = 'sound/vox_fem/away.ogg',
 "b" = 'sound/vox_fem/b.ogg',
 "back" = 'sound/vox_fem/back.ogg',
-"backman" = 'sound/vox_fem/backman.ogg',
 "bad" = 'sound/vox_fem/bad.ogg',
 "bag" = 'sound/vox_fem/bag.ogg',
-"bailey" = 'sound/vox_fem/bailey.ogg',
```
Comment (FlattestGuitar) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm]: Why

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 6]
```diff
@@ -17,22 +16,23 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
...
+"airlock" = 'sound/vox_fem/airlock.ogg',
 "alarm" = 'sound/vox_fem/alarm.ogg',
+"alcohol" = 'sound/vox_fem/alcohol.ogg',
 "alert" = 'sound/vox_fem/alert.ogg',
 "alien" = 'sound/vox_fem/alien.ogg',
 "aligned" = 'sound/vox_fem/aligned.ogg',
 "all" = 'sound/vox_fem/all.ogg',
 "alpha" = 'sound/vox_fem/alpha.ogg',
 "am" = 'sound/vox_fem/am.ogg',
-"amigo" = 'sound/vox_fem/amigo.ogg',
```
Comment (FlattestGuitar) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm]: Why

File: code/modules/mob/living/silicon/ai/vox_sounds.dm [hunk 7]
```diff
@@ -17,22 +16,23 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
 "across" = 'sound/vox_fem/across.ogg',
 "activate" = 'sound/vox_fem/activate.ogg',
 "activated" = 'sound/vox_fem/activated.ogg',
+"active" = 'sound/vox_fem/active.ogg',
 "activity" = 'sound/vox_fem/activity.ogg',
-"adios" = 'sound/vox_fem/adios.ogg',
```
Comment (FlattestGuitar) [inline code/modules/mob/living/silicon/ai/vox_sounds.dm]: Why

**Comment by EldritchSigma** [comment]
Clickbaity title.

Other than the removal of swearwords I like it. I mean an AI should call an asshole an asshole. Atleast no AIs saying "Captain got wood. Unathi required in Captain Office."

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 5 above)
Comment (SamHPurp) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: Tell me what this'll be used for.

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 4 above)
Comment (SamHPurp) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: Tell me what this'll be used for.

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 3 above)
Comment (SamHPurp) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: Tell me what this'll be used for.

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 1 above)
Comment (SamHPurp) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: Tell me what this'll be used for.

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 6 above)
Comment (SamHPurp) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: Unnecessary, especially when we tout "use English here"

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 7 above)
Comment (SamHPurp) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: Unnecessary, especially when we tout "use English here"

**Comment by EldritchSigma** [comment]
Also TIL AI can say DEEOO!!! Fucking JoJo.

**Comment by Spartan6** [comment]
Build me up, build me up, buttercup baby, just to let me down...

**Comment by FlattestGuitar** [comment]
Apologies for the late response.

I don't agree with removing "adios" and "amigo". Both of the words are commonly used in English speech and the AI going "adios amigo" as it spaces some nukies would be hilarious. I don't believe that the words being present in any way hints at the fact that we somehow allow the use of languages other than English to a wider extent.

The other words I brought up previously might as well go, I don't care as much.

**Comment by FlattestGuitar** [comment]
No amigo, no merge.

@AndrewMontagne, feel free to overrule me.

**Comment by SamHPurp** [comment]
Will re-add the Spanish when the baby goes back down to sleep.

**Comment by Ziiro** [comment]
adios

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 1 above)
Comment (SmokingKilz) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: "Wood frame detected in arrivals" Is a quick example. I don't think we should be removing stuff like that, instead just punish idiots that try and use it to recite WGW

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 4 above)
Comment (SmokingKilz) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: To say centimeter as AI?

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 1 above)
Comment (SamHPurp) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: frame isn't a word the AI can say.

File: code/modules/mob/living/silicon/ai/vox_sounds.dm (hunk 4 above)
Comment (SamHPurp) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: Canal to say centimeter? Okay? How?

Also why would an AI need to say Centimeter.

Comment (SmokingKilz) [reply code/modules/mob/living/silicon/ai/vox_sounds.dm]: Nevermind, I'm retarded. For some reason I read "Canal" as "centi'

**Pull request merged**